index.html

```diff
@@ -716,7 +766,7 @@ <h1>Streaming GCN<br>Notices in Python</h1>
         </div>
       </div>
     </section>
-
+  -->
```

Comment:
Don't comment it out, delete it!

Reply:
I was trying to see if I could fit the additional enhancement text, but ok.